src/elements/content-sidebar/__tests__/SidebarNav-test.js

```diff
@@ -5,7 +5,7 @@ import IconMagicWand from '../../../icons/general/IconMagicWand';
 import IconMetadataThick from '../../../icons/general/IconMetadataThick';
 import IconDocInfo from '../../../icons/general/IconDocInfo';
 import IconChatRound from '../../../icons/general/IconChatRound';
-import AdditionalTabs from '../AdditionalTabs';
+import AdditionalTabs from '../additional-tabs/AdditionalTabs';
```

Comment:
I think we've started to prefer directory style imports via index.js, so `import { AdditionalTabs } from '../additional-tabs';`